opentok/opentok-web-samples
# OPENTOK-49577: Update Electron and Webrtc-js versions for electron basic video chat sample app

Currently we use very old versions of electron and webrtc-js (2 and 2.16 respectively). This PR updates the versions to the latest stable releases of both. It also adds `nodeIntegration` and `contextIsolation` to the BrowserWindow initialization as the default values were changed in electron 5. 

To test: 

- Set credentials within openTok.js
- run `npm install && npm start`
- verify that video publishes correctly
